src/pages/documentation/adobe-analytics/release-notes.md

```diff
@@ -1,5 +1,16 @@
 # Release Notes
 
+## June 6, 2023
+
+### iOS Analytics 4.0.0
+
+Major version update for [Adobe Analytics](./index.md) for Adobe Experience Platform Mobile SDKs on iOS compatible with Mobile Core 4.0.0. The current release includes the following changes:
+
+* Updated the minimum supported version to iOS 11.0 and tvOS 11.0.
+* Include XCFrameworks built with Xcode 14.1 with the GitHub release.
+* Resolved an issue with retrieving post-processed data in Assurance for track calls with empty state.
+* Added `a.systemLocale` to Analytics hits when Lifecycle extension is used.
```

Comment:
maybe specify that it's only added on the lifecycle hit, not to all Analytics hits?